plugins/CoreAdminHome/Emails/TwoFactorAuthEnabledEmail.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\Emails;
+
+use Piwik\Piwik;
+use Piwik\Plugins\CoreAdminHome\Emails\SecurityNotificationEmail;
+
+class TwoFactorAuthEnabledEmail extends SecurityNotificationEmail
+{
+    /**
+     * @var string[]
+     */
+    private $superuserEmails;
+
+    public function __construct($login, $emailAddress, $superuserEmails)
+    {
+        $this->superuserEmails = $superuserEmails;
+
+        parent::__construct($login, $emailAddress);
+    }
+
+    protected function getBody()
+    {
+        $email = $this->getASuperUserEmail();
+
+        if (!$email) {
+            return Piwik::translate('CoreAdminHome_SecurityNotificationTwoFactorAuthEnabledBodyNoEmail');
+        }
+
+        return Piwik::translate('CoreAdminHome_SecurityNotificationTwoFactorAuthEnabledBody', [$email]);
+    }
+
+    private function getASuperUserEmail()
+    {
+        if (count($this->superuserEmails)) {
+            foreach ($this->superuserEmails as $login => $email) {
+                return $email;
+            }
+        }
```

Comment:
here it's possible we send the email to ourselves. should we find the first superuser email which is not us?
what if there is no other superuser? what is the recommendation in the email to the user?

Reply:
ping @tsteur / @mattab

Reply:
Not sure I understand, it should be sent to the user that enabled the two factor and nobody else?

Reply:
Like you can't enable it for a different user AFAIK?

Reply:
Oh i see now it's for who we mention to contact, I'll have a think

Reply:
Let's keep it simple and do same thing as for password change etc.
 We could reuse the same translation key : `UsersManager_IfThisWasYouPasswordChange`. which says `If this was you, feel free to ignore this email. If this was not you, please contact your Matomo administrator immediately, as your account might have been compromised!` and this would maybe also make the other comments obsolete. 

I wonder if we can document this for future reference somewhere but wouldn't really know where right now. It's really good to have this `SecurityNotificationEmail` base class now maybe this could provide such a method or constant or something to be reused across emails?

Reply:
I reused that translation key. I'll think about that method or constant.